server_ftp/models/server_ftp.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, exceptions, fields, models
+
+from ..lib.ftp_server import FtpServer
```

Comment:
You have to update these import to reflect the uppercased names.

This is causing the runbot failure:
```
  File "/home/odoo/build/OCA/server-tools/server_ftp/models/server_ftp.py", line 5, in <module>
    from ..lib.ftp_server import FtpServer
ImportError: cannot import name 'FtpServer'
```